daemon/cmd/hubble.go

```diff
@@ -332,3 +341,24 @@ func (d *Daemon) LookupSecIDByIP(ip net.IP) (id ipcache.Identity, ok bool) {
 func (d *Daemon) GetK8sStore(name string) k8scache.Store {
 	return d.k8sWatcher.GetStore(name)
 }
+
+// getHubbleEventBufferCapacity returns the user configured capacity for
+// Hubble's events buffer. The deprecated flag hubble-flow-buffer-size is
+// evaluated if greater than 0, otherwise the new flag
+// hubble-event-buffer-capacity is used instead.
+func getHubbleEventBufferCapacity(logger logrus.FieldLogger) (container.Capacity, error) {
+	// check deprecated old flag for compatibility
+	// TODO: remove support for HubbleFlowBufferSize once 1.11 is out
+	if option.Config.HubbleFlowBufferSize > 0 {
+		logger.Warningf("Option '%s' is deprecated and will be removed in Cilium 1.11", option.HubbleFlowBufferSize)
```

Comment:
Is this redundant given that the flag is already marked as deprecated in `daemon/cmd/daemon_main.go` ?